src/components/views/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,42 @@
+import {Text, View} from 'react-native';
+import Logo from '../../../static/imgs/rst_logo.svg';
+import {styles} from './style';
+import {BackArrow, LoginForm, Main, RegisterForm} from '../../../components';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {useNavigation} from '@react-navigation/native';
+import {globalStyles} from '../../../utils/globalStyles';
+import {useState} from 'react';
+import {ROUTES, RootStackParamList} from '../../../static/types/routeTypes';
+import {StackNavigationProp, createStackNavigator} from '@react-navigation/stack';
+
+const Tab = createNativeStackNavigator();
+
+export const MainPage = () => {
+  const [index, setIndex] = useState(0);
+  const navigation = useNavigation<StackNavigationProp<RootStackParamList>>();
+
+  return (
+    <View style={styles.main}>
+      {index !== 0 && <BackArrow goBack={() => navigation.navigate(ROUTES.MAIN)} />}
```

Comment:
Po prostu header do nawigacji możesz dodać od razu z tym back buttonem

Reply:
ale on wygląda chujowo i tytuł jest i zajmuje miejsce

Reply:
Można custom header normalnie zrobić w nawigacji, i wtedy masz po bożemu

Reply:
nie da rady chyba tego tak ogarnąć jak ty myślisz bo tu jest 2 nawigacja i header do niej będzie się wyświetlał nad nią a nie na samej górze czyli pod logiem a jak w tej głównej nawigacji to dam to wtedy wszędzie to będę miał i i tak będę potrzebował podobnego sposobu na rozpoznanie jaki to jest screen żeby tylko na tym mi się pojawiała strzałka